setup.py

```diff
@@ -1,29 +1,30 @@
 """A library of components for batou.
 """
 
-from setuptools import setup, find_packages
 import glob
 import os.path
 
+from setuptools import find_packages, setup
+
 
 def project_path(*names):
     return os.path.join(*names)
 
 
 setup(
     name='batou_ext',
-    version='0.1dev0',
+    version='2.3.dev0',
     install_requires=[
-        'batou>=2',
+        'batou>=2, <2.3',
```

Comment:
why <2.3? Is this the status intended to be useful for 2.2?

Reply:
Currently we test against the latest final version.
It is not possible to have a version which supports both 2.2 and 2.3 (so maybe 2.3 is actually a version 3) – Only after creating the PR I found out that I am on the 2.3 branch, so this has to be fixed.